setup.cfg

```diff
@@ -0,0 +1,101 @@
+[metadata]
+name = smarts
+version = 0.6.1
+url = https://github.com/huawei-noah/SMARTS
+long_description = file:utils/setup/README.pypi.md
+long_description_content_type = text/markdown
+license=MIT
+classifiers=
+    Development Status :: 5 - Production/Stable
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    License :: OSI Approved :: MIT License
+
+[options]
+packages = find:
+include_package_data = True
+zip_safe = True
+python_requires = >=3.7
+install_requires = 
```

Comment:
Ensure the requirements are updated with the latest requirements from the develop branch prior to merging.